main/src/ca/uwaterloo/flix/tools/SimpleRunner.scala

```diff
@@ -63,7 +63,8 @@ object SimpleRunner {
     }
 
     // check if we should start a REPL
-    if (cmdOpts.command == Command.None && cmdOpts.files.isEmpty) {
```

Comment:
Leave this unchanged. We will deal with `--test` separately.

Reply:
Ah, sorry, I misunderstood ('dealing with --test/--benchmark').